build.ps1

```diff
@@ -28,20 +28,29 @@ function CrossGen([string] $runtime, [bool] $isSelfContained, [string] $publishT
     $regular = Join-Path $publishTarget "regular"
     $crossGen = "$publishTarget\download\crossgen\crossgen.exe"
 
-    #DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.microsoft.netcore.jit/2.1.0-preview2-26316-09" @("runtimes\$runtime\native\clrjit.dll")  @("$publishTarget\download\clrjit")
-    #DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.Microsoft.NETCore.Runtime.CoreCLR/2.1.0-preview2-26316-09"  @("tools\crossgen.exe")  @("$publishTarget\download\crossgen")
+	#$symbolsPath = Join-Path $publishTarget "Symbols"
+    #new-item -itemtype directory -path $symbolsPath
 
-    DownloadNupkg "https://www.nuget.org/api/v2/package/runtime.$runtime.Microsoft.NETCore.Jit/2.0.5" @("runtimes\$runtime\native\clrjit.dll")  @("$publishTarget\download\clrjit")
-    DownloadNupkg "https://www.nuget.org/api/v2/package/runtime.$runtime.Microsoft.NETCore.Runtime.CoreCLR/2.0.5"  @("tools\crossgen.exe")  @("$publishTarget\download\crossgen")
+    DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.Microsoft.NETCore.Jit/2.1.0-rtm-26528-02" @("runtimes\$runtime\native\clrjit.dll")  @("$publishTarget\download\clrjit")
```

Comment:
👍 
Thought I had updated this with the change to 2.1. Likely a commit that was missed when the PR was broken up.